app/services/twilio_service.rb

```diff
@@ -0,0 +1,59 @@
+module TwilioService
+  class << self
+
+    def answer_call(params={})
+      @contact = params[:contact]
+      ::SlackWebClient.post_message(slack_call_message('ringing')) unless params[:From].include?('client')
+      ::TwilioTwiml.dial_twiml(params).to_xml
+    end
+
+    def send_to_voicemail(params={})
+      ::TwilioTwiml.voicemail_twiml(voicemail_message).to_xml if params['DialCallStatus'] == 'no-answer'
+    end
+
+    def post_slack_call(params={})
+      status ||= params['CallStatus']
+      ::SlackWebClient.post_message(slack_call_message(status)) unless params['Direction'] == 'outbound-dial'
+    end
+
+    def handle_voicemail_recording(params={})
+      url ||= params['RecordingUrl']
+      add_voicemail_link(url)
+      ::SlackWebClient.post_message(slack_recording_message(url))
+    end
+
+    def text_response(params={})
+      args = { sender: params[:From], body: params[:Body] }
+      ::SlackWebClient.post_message(slack_text_message(args))
+      ::TwilioTwiml.text_twiml(params).to_xml
+    end
+
+    private
+
+      def slack_call_message(status)
+        link = "#{Ngrok.web_hook_host}/contacts/#{@contact.id}"
+        message = {
```

Comment:
why not have the possible values for your status variable match the keys in your i18n file?